node_modules/oae-core/documentpreview/js/documentpreview.js

```diff
@@ -164,6 +164,17 @@ define(['jquery', 'underscore', 'oae.core', 'lazyload'], function($, _, oae) {
                     return callback({'code': jqXHR.status, 'msg': jqXHR.responseText});
                 }
             });
+
+            // Request the page content text version
+            $.ajax({
```

Comment:
It would probably be better to initiate the call to load the text version in the success block of the html version. That will ensure that our signature is always up to date. In order to do this cleanly, we should probably split up the `loadPage` function into a `renderPage` function that does the preparatory work, a `loadHTMLPage` function that loads the HTML page and a `loadTextPage` function that loads the text version.